php/class-fieldmanager-field.php

```diff
@@ -95,6 +95,20 @@ abstract class Fieldmanager_Field {
 	 */
 	public $label_after_element = false;
 
+	/**
+	 * Button content for the form element.
+	 *
+	 * @var string
+	 */
+	public $button_content = '';
```

Comment:
This should go into `class-fieldmanager-button.php`. Adding it here means that any field can set `button_text`, which isn't the case.